Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -17,6 +17,8 @@ public class PokePointer : BaseControllerPointer, IMixedRealityNearPointer
         private float closestDistance = 0.0f;
 
         private Vector3 closestNormal = Vector3.forward;
+        // previous frame pointer position
```

Comment:
Do you need to set prev position = cur position on first frame / on source detected?

Reply:
i didn't see any issues, but you're right I should do that!